core/src/main/kotlin/com/alecstrong/sql/psi/core/SqlLexer.flex

```diff
@@ -160,6 +160,8 @@ STRING=('([^'])*'|\"([^\"])*\")
   "GLOB"                 { return GLOB; }
   "REGEXP"               { return REGEXP; }
   "ESCAPE"               { return ESCAPE; }
+  "IS DISTINCT FROM"     { return IS_DISTINCT_FROM; }
```

Comment:
Don't use multiple tokens here, only create the missing tokens, if there are any missing at all. `IS`, `DISTINCT`, `FROM` and `NOT` should be already defined.